src/Faker/Provider/fa_IR/Internet.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Faker\Provider\fa_IR;
+
+class Internet extends \Faker\Provider\Internet
+{
+    protected static $userNameFormats = array(
+        '{{lastNameAscii}}.{{firstNameAscii}}',
+        '{{firstNameAscii}}.{{lastNameAscii}}',
+        '{{firstNameAscii}}##',
+        '?{{lastNameAscii}}',
+    );
+
+    protected static $safeEmailTld = array(
+        'com', 'ir', 'me', 'net', 'org',
+    );
+
+    protected static $tld = array(
+        'biz', 'com', 'info', 'ir', 'net', 'org',
+    );
+
+    protected static $lastNameAscii = array(
+        'ahmadi', 'rezaei', 'mohammady', 'salari', 'zare', 'chamran', 'hosseini', 'hasan', 'shahriari', 'karimi', 'majidi', 'moosavi', 'soleimani', 'ebrahimi', 'abbasi', 'majedi', 'nili', 'yazdi', 'jalili', 'babaei', 'ghasemi',
```

Comment:
A few dozen names at least. Look at the other providers for an example.

One question: can't translitteration from arabic work?